--- a/thruk.conf	2012-05-22 00:09:59.573802098 +0200
+++ b/thruk.conf	2012-05-22 00:11:32.960675966 +0200
@@ -62,11 +62,11 @@
 
 ######################################
 # location of your cgi.cfg
-cgi.cfg             = cgi.cfg
+cgi.cfg             = /etc/thruk/cgi.cfg
 
 ######################################
 # use frames, can be useful for addons
-use_frames          = 0
+use_frames          = 1
 
 ######################################
 # normally passive checks would be marked as disabled.
@@ -133,27 +133,27 @@
 
 ######################################
 # temporary directory, used for template cache...
-#tmp_path = /tmp
+tmp_path = /var/cache/thruk
 
 ######################################
 # var directory, used to store user settings
-#var_path = ./var
+var_path = /var/lib/thruk
 
 ######################################
 # server side includes
-#ssi_path = ssi/
+ssi_path = /etc/thruk/ssi/
 
 ######################################
 # path to your plugins
 # see documentation for further steps if
 # you change the plugin path
-#plugin_path = plugins/
+plugin_path = /etc/thruk/plugins/
 
 ######################################
 # path to your themes
 # see documentation for further steps if
 # you change the themes path
-#themes_path = themes/
+themes_path = /etc/thruk/themes/
 
 ######################################
 # custom templates
@@ -161,7 +161,7 @@
 
 ######################################
 # log4perl config
-#log4perl_conf = ./log4perl.conf
+log4perl_conf = /etc/thruk/log4perl.conf
 
 ######################################
 # use paged data instead of all data in one huge page
@@ -341,7 +341,7 @@
 # Specifiy a file which is then completly under the control of Thruk.
 # It will be used to store cronjobs, ex. for reports. The file has to
 # be writable by Thruk.
-#cron_file =
+cron_file = /etc/cron.d/thruk
 
 # The post edit cmd is necessary for OMD where you need to reload the
 # crontab after editing.
@@ -374,6 +374,20 @@
 ######################################
 # Backend Configuration, enter your backends here
 <Component Thruk::Backend>
+    <peer>
+        name   = Core
+        type   = livestatus
+        <options>
+            peer          = /var/nagios/rw/live
+            resource_file = /etc/nagios/resource.cfg
+       </options>
+       <configtool>
+            core_conf      = /etc/nagios/nagios.cfg
+            obj_check_cmd  = /usr/sbin/nagios -v /etc/nagios/nagios.cfg
+            obj_reload_cmd = /etc/init.d/nagios reload
+       </configtool>
+    </peer>
+
 #    <peer>
 #        name   = Local Nagios
 #        type   = livestatus
@@ -423,7 +437,7 @@
 <Component Thruk::Plugin::ConfigTool>
     show_plugin_syntax_helper = 1
 
-    thruk    = ./thruk_local.conf
-    cgi.cfg  = ./cgi.cfg
-#    htpasswd = ./htpasswd
+    thruk    = /etc/thruk/thruk_local.conf
+    cgi.cfg  = /etc/thruk/cgi.cfg
+    htpasswd = /etc/thruk/htpasswd
 </Component>
