--- a/log4perl.conf.example	2012-02-08 14:07:36.785219995 +0100
+++ b/log4perl.conf.example	2012-02-08 14:55:26.540019880 +0100
@@ -14,7 +14,7 @@
 
 # normal log messages
 log4perl.appender.ThrukLog=Log::Dispatch::File
-log4perl.appender.ThrukLog.filename=logs/error.log
+log4perl.appender.ThrukLog.filename=/var/log/thruk/error.log
 log4perl.appender.ThrukLog.mode=append
 log4perl.appender.ThrukLog.Threshold=INFO
 log4perl.appender.ThrukLog.layout=Log::Log4perl::Layout::PatternLayout
