--- a/lib/Thruk.pm	2012-02-12 19:16:32.895854619 +0100
+++ b/lib/Thruk.pm	2012-02-12 19:19:03.412544661 +0100
@@ -43,6 +43,7 @@
 ###################################################
 # Configure the application.
 #
+__PACKAGE__->config->{home} = "/usr/share/thruk";
 # Note that settings in thruk.conf (or other external
 # configuration file that you set up manually) take precedence
 # over this when using ConfigLoader. Thus configuration
