--- a/fcgid_env.sh	2014-02-17 13:51:42.604007060 -0500
+++ b/fcgid_env.sh	2014-02-18 09:50:22.631111321 -0500
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # set omd environment
-export CATALYST_CONFIG="/etc/thruk"
+export CATALYST_CONFIG="@SYSCONFDIR@"
 
 # load extra environment variables
 if [ -f /etc/sysconfig/thruk ]; then
@@ -13,4 +13,4 @@
 [ -e ~/.thruk ] && . ~/.thruk
 
 # execute fastcgi server
-exec /usr/share/thruk/script/thruk_fastcgi.pl
+exec @DATADIR@/script/thruk_fastcgi.pl
--- a/script/thruk_fastcgi.pl	2014-02-17 13:51:42.602007012 -0500
+++ b/script/thruk_fastcgi.pl	2014-02-18 09:49:13.206496762 -0500
@@ -3,6 +3,9 @@
 use strict;
 use warnings;
 
+use lib '@THRUKLIBS@';
+use lib '@DATADIR@/lib';
+
 ###################################################
 # create connection pool
 # has to be done really early to save memory
--- a/thruk_auth	2014-02-17 13:51:42.602007012 -0500
+++ b/thruk_auth	2014-02-18 09:49:13.206496762 -0500
@@ -12,8 +12,9 @@
   export PERL5LIB=$OMD_ROOT/share/thruk/lib:$PERL5LIB
   if [ -z $CATALYST_CONFIG ]; then export CATALYST_CONFIG="$OMD_ROOT/etc/thruk"; fi
 else
-  export PERL5LIB=$PERL5LIB:/usr/share/thruk/lib:/usr/lib/thruk/perl5;
-  if [ -z $CATALYST_CONFIG ]; then export CATALYST_CONFIG='/etc/thruk'; fi
+  export PERL5LIB=$PERL5LIB:@DATADIR@/lib:@THRUKLIBS@;
+  if [ -z $CATALYST_CONFIG ]; then export CATALYST_CONFIG='@SYSCONFDIR@'; fi
+  if [ -z $REMOTEURL ]; then export REMOTEURL='http://localhost@HTMLURL@/cgi-bin/remote.cgi'; fi
 fi
 
 eval 'exec perl -x $0 ${1+"$@"} ;'
