--- a/script/thruk_fastcgi.pl	2012-02-09 02:00:15.888835121 +0100
+++ b/script/thruk_fastcgi.pl	2012-02-09 20:49:33.725412141 +0100
@@ -1,21 +1,10 @@
 #!/usr/bin/perl
 
+use lib '/usr/lib/thruk/perl5';
+use lib '/usr/share/thruk/lib';
 use strict;
 use warnings;
 
-BEGIN {
-    use FindBin;
-    use Config;
-    if(-e $FindBin::Bin."/../local-lib") {
-        ### use 5.10.0; # only required for packages ###
-        unshift @INC, $FindBin::Bin."/../local-lib/lib/perl5";
-        if(! -e $FindBin::Bin."/../local-lib/lib/perl5/".$Config{archname}) {
-            die("\nERROR: this is the wrong precompiled version, your archname is: ".$Config{archname}."\n\n");
-        }
-        require local::lib; local::lib->import($FindBin::Bin."/../local-lib/perl5/");
-    }
-}
-
 use Catalyst::ScriptRunner;
 Catalyst::ScriptRunner->run('Thruk', 'FastCGI');
 
