--- a/thruk.conf	2014-01-22 23:16:18.000000000 +0100
+++ b/thruk.conf	2014-01-22 23:18:38.000000000 +0100
@@ -15,6 +15,9 @@
 # set the url prefix for all urls
 url_prefix = /
 
+# product prefix must be used if thruk is not deployed under /thruk/...
+product_prefix = @HTMLURL@
+
 ######################################
 # Changes the path to your logo images. Default is
 # $url_prefix+'thruk/themes/'+$current_theme+'/images/logos/' and
@@ -43,7 +46,7 @@
 # set the title prefix for all urls
 # this piece of text will be prepended to
 # all page titles.
-#title_prefix = Prod
+title_prefix = Naemon - 
 
 ######################################
 # uses bookmark names as page titles where possible
@@ -190,8 +193,8 @@
 
 ######################################
 # mode used when creating or saving files
-#mode_file = 0660
-#mode_dir  = 0770
+mode_file = 0660
+mode_dir  = 0770
 
 ######################################
 # files owners for new / changed files
@@ -216,7 +219,7 @@
 
 ######################################
 # custom templates
-#user_template_path = ./my_templates
+user_template_path = @SYSCONFDIR@/user_templates/
 
 ######################################
 # log4perl config
@@ -520,7 +523,7 @@
 # bug reporting
 # show link to bug reports when internal errors occur.
 # set to 'server' to log js error server side
-show_error_reports = 1
+show_error_reports = server
 
 # don't report some known harmless javascript errors
 skip_js_errors  = cluetip is not a function
@@ -577,56 +580,19 @@
 ######################################
 # Backend Configuration, enter your backends here
 <Component Thruk::Backend>
-#    <peer>
-#        name   = Core
-#        type   = livestatus
-#        <options>
-#            peer          = /var/naemon/rw/live
-#            resource_file = /etc/naemon/resource.cfg
-#       </options>
-#       <configtool>
-#            core_conf      = /etc/naemon/naemon.cfg
-#            obj_check_cmd  = /usr/bin/naemon -v /etc/naemon/naemon.cfg
-#            obj_reload_cmd = /etc/init.d/naemon reload
-#       </configtool>
-#    </peer>
-
-#    <peer>
-#        name   = Local Naemon
-#        type   = livestatus
-#        hidden = 1             # make this backend hidden by default
-#        groups = admins        # make this backend only visible to the admin group
-#        <options>
-#            peer          = /tmp/livestatus.socket
-#            resource_file = path_to_your/resource.cfg
-#       </options>
-#       <configtool>
-#            core_conf      = /etc/naemon/naemon.cfg
-#            obj_check_cmd  = /etc/init.d/naemon checkconfig
-#            obj_reload_cmd = /etc/init.d/naemon reload
-#       </configtool>
-#    </peer>
-#    <peer>
-#        name   = External Naemon
-#        type   = livestatus
-#        <options>
-#            peer    = 172.16.0.1:9999
-#       </options>
-#    </peer>
-#    <peer>
-#        name   = External Icinga
-#        type   = livestatus
-#        <options>
-#            peer    = 172.16.0.2:9999
-#       </options>
-#    </peer>
-#    <peer>
-#        name   = External Shinken
-#        type   = livestatus
-#        <options>
-#            peer    = 127.0.0.01:50000
-#       </options>
-#    </peer>
+    <peer>
+        name   = Naemon
+        type   = livestatus
+        <options>
+            peer          = /var/cache/naemon/live
+            resource_file = @SYSCONFDIR@/resource.cfg
+       </options>
+       <configtool>
+            core_conf      = @SYSCONFDIR@/naemon.cfg
+            obj_check_cmd  = @BINDIR@/naemon -v @SYSCONFDIR@/naemon.cfg
+            obj_reload_cmd = @INITDIR@/naemon reload
+       </configtool>
+    </peer>
 </Component>
 
 
@@ -714,21 +680,21 @@
 <Component Thruk::Plugin::BP>
     # Results will be send back by using the spool folder.
     # This folder should point to the 'check_result_path' of your core.

-    #spool_dir = /var/naemon/rw/check_results
+    spool_dir = @CHECKRESULTDIR@
 
     # as alternative to the spool_dir, a livestatus connection can be used
     # to send results to the core.
-    #result_backend = sitename
+    result_backend = Naemon
 
     # Save objects to this file. Content will be overwritten.
-    #objects_save_file = /etc/naemon/conf.d/thruk_bp_generated.cfg
+    objects_save_file = @SYSCONFDIR@/conf.d/thruk_bp_generated.cfg
 
     # User maintained file containing templates used for business process services.
-    #objects_templates_file = /etc/naemon/conf.d/thruk_templates.cfg
+    objects_templates_file = @SYSCONFDIR@/conf.d/thruk_templates.cfg
 
     # Command to apply changes to the objects_save_file. If no set and
     # and result_backend is used, the restart will be triggered as external command.
-    #objects_reload_cmd = /etc/init.d/naemon reload
+    #objects_reload_cmd = @INITDIR@/naemon reload
 
     # hooks which will be executed before or after saving.
     #pre_save_cmd   =
