--- a/script/thruk	2012-02-19 19:43:14.636750876 +0100
+++ b/script/thruk	2012-02-19 19:44:57.731751222 +0100
@@ -3,18 +3,8 @@
 use strict;
 use warnings;
 
-BEGIN {
-    use FindBin;
-    use Config;
-    if(-e $FindBin::Bin."/../local-lib") {
-        ### use 5.10.0; # only required for packages ###
-        unshift @INC, $FindBin::Bin."/../local-lib/lib/perl5";
-        if(! -e $FindBin::Bin."/../local-lib/lib/perl5/".$Config{archname}) {
-            die("\nERROR: this is the wrong precompiled version, your archname is: ".$Config{archname}."\n\n");
-        }
-        require local::lib; local::lib->import($FindBin::Bin."/../local-lib/perl5/");
-    }
-}
+use lib '/usr/share/thruk/lib';
+use lib '/usr/lib/thruk/perl5';
 
 # set necessary env values
 if(defined $ARGV[0]) {
