--- a/script/thruk	2012-02-20 12:57:03.364764756 +0100
+++ b/script/thruk	2012-02-20 12:58:45.526016418 +0100
@@ -3,18 +3,11 @@
 use strict;
 use warnings;
 
+use lib '/usr/lib/thruk/perl5';
+use lib '/usr/share/thruk/lib';
 BEGIN {
-    use FindBin;
-    use Config;
-    if(-e $FindBin::Bin."/../local-lib") {
-        ### use 5.10.0; # only required for packages ###
-        unshift @INC, $FindBin::Bin."/../local-lib/lib/perl5";
-        if(! -e $FindBin::Bin."/../local-lib/lib/perl5/".$Config{archname}) {
-            die("\nERROR: this is the wrong precompiled version, your archname is: ".$Config{archname}."\n\n");
-        }
-        require local::lib; local::lib->import($FindBin::Bin."/../local-lib/perl5/");
-    }
-}
+    $ENV{'CATALYST_CONFIG'} = '/etc/thruk' unless defined $ENV{'CATALYST_CONFIG'};
+};
 
 ##############################################
 
