--- a/script/thruk	2012-08-19 10:32:11.428007339 +0200
+++ b/script/thruk	2012-08-19 10:32:20.679098831 +0200
@@ -9,6 +9,12 @@
 #! -*- perl -*-
 # vim: expandtab:ts=4:sw=4:syntax=perl
 
+BEGIN {
+    use lib '/usr/lib/thruk/perl5';
+    use lib '/usr/share/thruk/lib';
+    $ENV{'CATALYST_CONFIG'} = '/etc/thruk' unless defined $ENV{'CATALYST_CONFIG'};
+};
+
 use strict;
 use warnings;
 use Getopt::Long;
