--- a/fcgid_env.sh	2013-12-03 23:11:49.748092221 +0100
+++ b/fcgid_env.sh	2013-12-03 23:12:52.971099121 +0100
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # set thruk environment
-export THRUK_CONFIG="/etc/thruk"
+export THRUK_CONFIG="@SYSCONFDIR@"
 
 # load extra environment variables
 if [ -f /etc/sysconfig/thruk ]; then
@@ -13,4 +13,4 @@
 [ -e ~/.thruk ] && . ~/.thruk
 
 # execute fastcgi server
-exec /usr/share/thruk/script/thruk_fastcgi.pl
+exec @DATADIR@/script/thruk_fastcgi.pl
