--- a/thruk.conf	2014-01-25 18:39:13.510437546 +0100
+++ b/thruk.conf	2014-01-26 16:42:44.506291579 +0100
@@ -2,7 +2,7 @@
   AddHandler fcgid-script .sh
   IPCCommTimeout 120
 
-  <Directory /usr/share/thruk>
+  <Directory @DATADIR@>
     Options FollowSymLinks
     AllowOverride All
     order allow,deny
@@ -12,7 +12,7 @@
       Require all granted
     </IfModule>
   </Directory>
-  <Directory /etc/thruk/themes>
+  <Directory @SYSCONFDIR@/themes>
     Options FollowSymLinks
     allow from all
     # required for apache 2.4
@@ -20,7 +20,7 @@
       Require all granted
     </IfModule>
   </Directory>
-  <Directory /etc/thruk/plugins>
+  <Directory @SYSCONFDIR@/plugins>
     Options FollowSymLinks
     allow from all
     # required for apache 2.4
@@ -29,33 +29,53 @@
     </IfModule>
   </Directory>
 
-  # redirect to a startup page when there is no pidfile yet
-  RewriteEngine On
-  RewriteCond %{REQUEST_METHOD} GET
-  RewriteCond %{REQUEST_URI} !^/thruk/startup.html
-  RewriteCond %{REQUEST_URI} !^/thruk/side.html
-  RewriteCond %{REQUEST_URI} !^/thruk/.*\.(css|png|js)
-  RewriteCond %{REQUEST_URI} ^/thruk
-  RewriteCond /var/cache/thruk/thruk.pid !-f
-  RewriteRule ^(.*)$ /thruk/startup.html?$1 [R=302,L,NE,QSA]
-
-  Alias /thruk/documentation.html /usr/share/thruk/root/thruk/documentation.html
-  Alias /thruk/startup.html /usr/share/thruk/root/thruk/startup.html
-  AliasMatch ^/thruk/(.*\.cgi|.*\.html)  /usr/share/thruk/fcgid_env.sh/thruk/$1
-  AliasMatch ^/thruk/plugins/(.*?)/(.*)$  /etc/thruk/plugins/plugins-enabled/$1/root/$2
-  Alias /thruk/themes/  /etc/thruk/themes/themes-enabled/
-  Alias /thruk /usr/share/thruk/root/thruk
+  Alias @HTMLURL@/documentation.html @DATADIR@/root/thruk/documentation.html
+  Alias @HTMLURL@/startup.html @DATADIR@/root/thruk/startup.html
+  AliasMatch ^@HTMLURL@/(.*\.cgi|.*\.html)  @DATADIR@/fcgid_env.sh@HTMLURL@/$1
+  AliasMatch ^@HTMLURL@/plugins/(.*?)/(.*)$  @SYSCONFDIR@/plugins/plugins-enabled/$1/root/$2
+  Alias @HTMLURL@/themes/  @SYSCONFDIR@/themes/themes-enabled/
+  Alias @HTMLURL@ @DATADIR@/root/thruk
 
-  <Location /thruk>
+  <Location @HTMLURL@>
     Options ExecCGI FollowSymLinks
-    AuthName "Thruk Monitoring"
+  </Location>
+  <Location @HTMLURL@/cgi-bin/restricted.cgi>
+    AuthName "Naemon Monitoring"
     AuthType Basic
-    AuthUserFile /etc/thruk/htpasswd
+    AuthUserFile /etc/naemon/htpasswd
     Require valid-user
   </Location>
-  <Location /thruk/cgi-bin/remote.cgi>
-    Order Deny,Allow
-    Allow from all
-    Satisfy any
-  </Location>
 </IfModule>
+
+RewriteEngine On
+<IfModule !mod_authz_core.c>
+  RewriteLock "@TMPDIR@/apache_rewrite.lock"
+</IfModule>
+
+<VirtualHost *:80>
+  # extend default virtual host. put/include these rewrite rules in ssl virtual host if you use ssl
+
+  # redirect to a startup page when there is no pidfile yet
+  RewriteCond %{REQUEST_METHOD}        GET
+  RewriteCond %{REQUEST_URI}           !^@HTMLURL@/startup.html
+  RewriteCond %{REQUEST_URI}           !^@HTMLURL@/side.html
+  RewriteCond %{REQUEST_URI}           !^@HTMLURL@/.*\.(css|png|js)
+  RewriteCond %{REQUEST_URI}           ^@HTMLURL@
+  RewriteCond @TMPDIR@/thruk.pid       !-f
+  RewriteRule ^(.*)$                   @HTMLURL@/startup.html?$1 [R=302,L,NE,QSA]
+
+  # redirect /thruk to new url, as the pkgs conflict anyway, this help migrating...
+  RewriteRule ^/thruk(.*)$             @HTMLURL@$1 [R=302,L]
+
+  # cookie based authentication
+  RewriteEngine On
+  RewriteMap  thruk_users              prg:@DATADIR@/thruk_auth
+  RewriteCond %{REQUEST_URI}           !^@HTMLURL@/cgi-bin/restricted.cgi
+  RewriteCond %{REQUEST_URI}           ^@HTMLURL@
+  RewriteCond %{HTTP_COOKIE}           (thruk_auth=[^;]+|$)  [NC]
+  RewriteRule ^/(.*)$                  /%1/%{REMOTE_ADDR}/____/$1/____/%{QUERY_STRING} [C,NS]
+  RewriteRule ^(.*)$                   ${thruk_users:$1|/loginbad/} [C,NS]
+  RewriteRule ^/pass/(.*)$             /$1 [NS,PT,L,E=!REMOTE_USER]
+  RewriteRule ^/redirect/(.*)$         /$1 [NS,L,R=302]
+  RewriteRule ^/loginok/([^/]+)/(.*)$  /$2 [NS,PT,L,E=REMOTE_USER:$1]
+</VirtualHost>
