--- a/cgi.cfg	2013-11-26 15:27:02.000000000 +0100
+++ b/cgi.cfg	2014-02-06 17:29:08.248728701 +0100
@@ -71,7 +71,7 @@
 # authorize any user who has authenticated to the web server.
 # Alternatively you can specify contactgroups too
 
-authorized_for_system_information=thrukadmin
+authorized_for_system_information=admin
 authorized_contactgroup_for_system_information=
 
 
@@ -84,7 +84,7 @@
 # to the web server.
 # Alternatively you can specify contactgroups too
 
-authorized_for_configuration_information=thrukadmin
+authorized_for_configuration_information=admin
 authorized_contactgroup_for_configuration_information=
 
 
@@ -98,7 +98,7 @@
 # authenticated to the web server.
 # Alternatively you can specify contactgroups too
 
-authorized_for_system_commands=thrukadmin
+authorized_for_system_commands=admin
 authorized_contactgroup_for_system_commands=
 
 
@@ -111,10 +111,10 @@
 # to authorize any user who has authenticated to the web server.
 # Alternatively you can specify contactgroups too
 
-authorized_for_all_services=thrukadmin
+authorized_for_all_services=admin
 authorized_contactgroup_for_all_services=
 
-authorized_for_all_hosts=thrukadmin
+authorized_for_all_hosts=admin
 authorized_contactgroup_for_all_hosts=
 
 
@@ -128,10 +128,10 @@
 # user who has authenticated to the web server.
 # Alternatively you can specify contactgroups too
 
-authorized_for_all_service_commands=thrukadmin
+authorized_for_all_service_commands=admin
 authorized_contactgroup_for_all_service_commands=
 
-authorized_for_all_host_commands=thrukadmin
+authorized_for_all_host_commands=admin
 authorized_contactgroup_for_all_host_commands=
 
 # READ-ONLY USERS
