From 8023d04a2d99d8bacbee4b5131b79250edd544b8 Mon Sep 17 00:00:00 2001
From: Sven Nierlein <sven@nierlein.de>
Date: Wed, 30 Dec 2009 14:50:50 +0100
Subject: [PATCH] added recovery state, fixed hidden external commands


Signed-off-by: Sven Nierlein <sven@nierlein.de>
---
 livestatus/src/LogEntry.cc |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/livestatus/src/LogEntry.cc b/livestatus/src/LogEntry.cc
index 3e71abe..466c66e 100644
--- a/livestatus/src/LogEntry.cc
+++ b/livestatus/src/LogEntry.cc
@@ -166,7 +166,6 @@ bool LogEntry::handleExternalCommandEntry(char *text, char *scan)
     if (!strcmp(text, "EXTERNAL COMMAND"))
     {
 	_logclass = LOGCLASS_COMMAND;
-	_text = _text + 18;
 	return true; // TODO: join with host/service information?
 	/* Damit wir die restlichen Spalten ordentlich befuellen, braeuchten
 	   wir eine komplette Liste von allen external commands und
@@ -187,6 +186,7 @@ int LogEntry::serviceStateToInt(char *s)
     // WARN, CRIT, OK, UNKNOWN
     switch (s[0]) {
 	case 'O': return 0;
+	case 'R': return 0;
 	case 'W': return 1;
 	case 'C': return 2;
 	case 'U': return 3;
@@ -197,9 +197,10 @@ int LogEntry::serviceStateToInt(char *s)
 
 int LogEntry::hostStateToInt(char *s)
 {
-    // WARN, CRIT, OK, UNKNOWN
+    // WARN, CRIT, OK, UNKNOWN, RECOVERY
     switch (s[1]) {
 	case 'P': return 0;
+	case 'E': return 0;
 	case 'O': return 1;
 	case 'N': return 2;
 	default: return 3;
-- 
1.6.0.4

